sympy/solvers/diophantine.py

```diff
@@ -1200,28 +1200,15 @@ def special_diop_DN(D, N):
     Usage
     =====
 
-    ``special_diop_DN(D, N)``: D and N are integers as in `x^2 - Dy^2 = N`.
+    WARNING: Internal method. Do not call directly!
+
+    ``_special_diop_DN(D, N)``: D and N are integers as in `x^2 - Dy^2 = N`.
 
     Details
     =======
 
     ``D`` and ``N`` correspond to D and N in the equation.
 
-    Examples
```

Comment:
Please leave the docstring. It's helpful for anyone wanting to use this function. It will be checked by the doctests. You just import the `_special_diop_DN`.